angular-workspace/projects/example-client-app/src/app/customapp/customapp.component.ts

```diff
@@ -92,6 +86,7 @@ export class CustomAppComponent {
     public onAddTableRow(): void {
```

Comment:
The behavior of the Blazor and Angular example apps for the Table are diverging pretty far from eachother. Can you bring them in sync? An example would be starting with zero items in both tables and then adding items with the button. I'd also appreciate adding many items on the button press instead of one, maybe 1K or so? And if possible having them be better table data to interact with (multiple items in groups)

Reply:
We have #978 to track the differences between Blazor & Angular, which have now diverged even more in this PR. My preference would be tackling that issue as a follow-up that I prioritize after this goes in. Does that seem reasonable, or do you think it should be done with this change?

Reply:
That seems reasonable, would be easier to compare of the edits to angular and blazor examples apps are in the same PR